### ISSUE TITLE
check that the twin BD tree creation makes sense

### ISSUE THREAD

**Issue by Giappo** [richelbilderbeek/razzo#53]
check the twin bd tree

**Comment by richelbilderbeek** [comment]
Yup, will take a look later, as I think there are higher priorities now. Moved this Issue to the 'To Do' column due to this.

**Comment by Giappo** [comment]
Mmm, it is a possibility. It's not strictly required just to raise the full architecture. It's definitely a problem to keep an eye on, though.

**Comment by richelbilderbeek** [comment]
I suggest to close this Issue for now, take a detailed look in the next phase of the Project. Closing :rainbow:

**Comment by richelbilderbeek** [comment]
Moved to Launch!
